pkg/controller/garbagecollector/patch.go

```diff
@@ -72,6 +72,15 @@ func (gc *GarbageCollector) getMetadata(apiVersion, kind, namespace, name string
 	return meta.Accessor(obj)
 }
 
+type objectForFinalizersPatch struct {
```

Comment:
clever

Reply:
I read this as both positive and negative, as I am sure it was intended.